contracts/atm/ATMLiquidityMiningInterface.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+/**
+    @notice This interface defines all function to interact with Liquidity Mining program.
+
+    @author develop@teller.finance
+ */
+interface ATMLiquidityMiningInterface {
+
```

Comment:
I think there are functions that are not in the interface (see the ATMLiquidityMining contract).

Reply:
New functions were added to the interface too.